### PR TITLE
Update TLDS to the Jakarta EE10 namespace (with TLDDoc Updates)

### LINKED ISSUE #179: Update Namespaces For TLDs for EE9/EE10
Identified after #172 ( Update to URIs) 

Should update to use 
```
<taglib
    xmlns="https://jakarta.ee/xml/ns/jakartaee"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-jsptaglibrary_3_0.xsd"
    version="3.0">
```

Currently, we can't update the TLD namespaces due to TLDDoc errors.

```
[INFO] --- exec-maven-plugin:3.0.0:java (generateTldDoc) @ jakarta.servlet.jsp.jstl ---
Loading and translating 5 Tag Libraries...
com.sun.tlddoc.GeneratorException: Error: /Users/siedlecki/open-source/jstl-api/impl/src/main/resources/META-INF/x.tld does not have xmlns="http://java.sun.com/xml/ns/javaee"
        at com.sun.tlddoc.TLDDocGenerator.createTLDSummaryDoc(TLDDocGenerator.java:555)
        at com.sun.tlddoc.TLDDocGenerator.generate(TLDDocGenerator.java:440)
        at com.sun.tlddoc.TLDDoc.main(TLDDoc.java:217)
        at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:254)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

### PR DESCRIPTION
**Pull request by volosied**
Fixes #179

- Update TLDs to Jakarta EE Namespace
- Update TLDDoc to generate documentation for the updated TLDs.
- The xsl updates were necessary as the tlddoc output would be empty. There' a lot I don't know, but, I found this to be the minimum to generate the proper documentation.  Please test to ensure everything looks good.

### CODE REVIEW

File: impl/src/main/resources/META-INF/c.tld [hunk 1]
```diff
@@ -17,10 +18,9 @@
 
 -->
 
-<taglib xmlns="http://java.sun.com/xml/ns/javaee"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-jsptaglibrary_2_1.xsd"
-    version="2.1">
+ <taglib xmlns="https://jakarta.ee/xml/ns/jakartaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-jsptaglibrary_3_0.xsd" version="3.0">
+
```
Comment (pnicolucci) [inline impl/src/main/resources/META-INF/c.tld:22]: Can you fix the formatting here so it's not all one line in these tlds? Would make reviewing a bit easier!

**Comment by pnicolucci** [comment]
@volosied I pulled down this branch and ran the following: `mvn clean install -Pstaging -Ptlddocs`

I got the following results:

> [INFO] --- exec-maven-plugin:3.0.0:java (generateTldDoc) @ jakarta.servlet.jsp.jstl ---
> Loading and translating 5 Tag Libraries...
> Generating docs for JSTL sql...
> [Fatal Error] :364:2: The content of elements must consist of well-formed character data or markup.
> ERROR:  'Could not compile stylesheet'
> FATAL ERROR:  'The content of elements must consist of well-formed character data or markup.'
>            :The content of elements must consist of well-formed character data or markup.
> com.sun.tlddoc.GeneratorException: javax.xml.transform.TransformerConfigurationException: The content of elements must consist of well-formed character data or markup.
>         at com.sun.tlddoc.TLDDocGenerator.generate(TLDDocGenerator.java:456)
>         at com.sun.tlddoc.TLDDoc.main(TLDDoc.java:217)
>         at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:254)
>         at java.base/java.lang.Thread.run(Thread.java:831)
> Caused by: javax.xml.transform.TransformerConfigurationException: The content of elements must consist of well-formed character data or markup.
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl.newTemplates(TransformerFactoryImpl.java:1059)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl.newTransformer(TransformerFactoryImpl.java:817)
>         at com.sun.tlddoc.TLDDocGenerator.generatePage(TLDDocGenerator.java:1183)
>         at com.sun.tlddoc.TLDDocGenerator.generateTagDetail(TLDDocGenerator.java:1118)
>         at com.sun.tlddoc.TLDDocGenerator.generateTLDDetail(TLDDocGenerator.java:1065)
>         at com.sun.tlddoc.TLDDocGenerator.generate(TLDDocGenerator.java:443)
>         ... 3 more
> Caused by: org.xml.sax.SAXParseException; lineNumber: 364; columnNumber: 2; The content of elements must consist of well-formed character data or markup.
>         at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1243)
>         at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.compiler.Parser.parse(Parser.java:429)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.compiler.Parser.parse(Parser.java:512)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.compiler.XSLTC.compile(XSLTC.java:476)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.compiler.XSLTC.compile(XSLTC.java:570)
>         at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl.newTemplates(TransformerFactoryImpl.java:1018)
>         ... 8 more
>

File: impl/src/main/resources/META-INF/sql.tld [hunk 2]
```diff
@@ -17,10 +18,9 @@
 
 -->
 
-<taglib xmlns="http://java.sun.com/xml/ns/j2ee"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
-    version="2.0">
+ <taglib xmlns="https://jakarta.ee/xml/ns/jakartaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-jsptaglibrary_3_0.xsd" version="3.0">
+
```
Comment (pnicolucci) [inline impl/src/main/resources/META-INF/sql.tld:22]: Please fix the formatting of this one as well. Thanks!

File: impl/src/main/resources/META-INF/c.tld [hunk 3]
```diff
@@ -17,10 +18,10 @@
 
 -->
 
-<taglib xmlns="http://java.sun.com/xml/ns/javaee"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-jsptaglibrary_2_1.xsd"
-    version="2.1">
+<taglib xmlns="https://jakarta.ee/xml/ns/jakartaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-jsptaglibrary_3_0.xsd"
```
Comment (pnicolucci) [inline impl/src/main/resources/META-INF/c.tld:23]: You seem to have a space at the end of this line throughout the tld changes you made. Let's clean that up.

File: tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl [hunk 4]
```diff
@@ -338,55 +338,63 @@
...
             <xsl:otherwise>false</xsl:otherwise>
           </xsl:choose>
         </td>        
         <td>
           <xsl:choose>
-            <xsl:when test="javaee:deferred-value">
+            <xsl:when test="jakartaee:deferred-value">
                 <xsl:choose>
+<<<<<<< HEAD
                     <xsl:when test="javaee:deferred-value/javaee:type">
```
Comment (pnicolucci) [inline tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl:365]: My guess is this is your issue with the error I'm getting locally, looks like a potential merge issue wasn't resolved.

File: tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl [hunk 5]
```diff
@@ -338,55 +338,63 @@
...
                         <code>jakarta.el.ValueExpression</code>
                         <br/>(<i>must evaluate to </i><code>java.lang.Object</code>)
                     </xsl:otherwise>
                 </xsl:choose>                                
             </xsl:when>
-            <xsl:when test="javaee:deferred-method">
+            <xsl:when test="jakartaee:deferred-method">
                 <xsl:choose>
+<<<<<<< HEAD
                     <xsl:when test="javaee:deferred-method/javaee:method-signature">
```
Comment (pnicolucci) [inline tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl:382]: Same issue here, this needs to be cleaned up.

File: impl/src/main/resources/META-INF/sql.tld (hunk 2 above)
Comment (volosied) [reply impl/src/main/resources/META-INF/sql.tld]: Too many tlds at this point!  Haha

File: tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl (hunk 4 above)
Comment (volosied) [reply tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl]: Fixed. Thanks for catching this.

**Comment by pnicolucci** [comment]
@volosied  I'm working to pull this branch down locally and test with it.

File: api/src/main/java/jakarta/servlet/jsp/jstl/tlv/package.html [hunk 6]
```diff
@@ -32,17 +32,17 @@
 <pre>
```
Comment (pnicolucci) [inline api/src/main/java/jakarta/servlet/jsp/jstl/tlv/package.html:32]: copyright update?

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**